homeassistant/components/media_player/bluesound.py

```diff
@@ -667,15 +808,99 @@ def supported_features(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        sleep = 0
+        if self._status is not None:
+            sleep = self._status.get('sleep', 0)
+
         return {
             ATTR_MODEL: self._model,
```

Comment:
Please remove model and brand info. That info does not belong in the state machine.